appinventor/components/src/com/google/appinventor/components/common/ComponentDescriptorConstants.java

```diff
@@ -0,0 +1,36 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
```

Comment:
For copyrights, you can remove Google as this is a new file, and the year for MIT should be 2019.